sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/View.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.view;
+
+import com.google.auto.value.AutoValue;
+import io.opentelemetry.sdk.metrics.aggregator.AggregatorFactory;
+import javax.annotation.concurrent.Immutable;
+
+@AutoValue
+@Immutable
+public abstract class View {
```

Comment:
If we decide to go with this structure, this class definitely needs javadoc, even if only placeholder javadoc for now.

Reply:
makes sense. Any ideas as to what mention there? My proposal
/**
* Encompasses aggregation and processing logic related to a specific view.
**/

Reply:
That seems a little recursive. What's a view?

Reply:
alright, I looked how OpenCensus defined it and here's what we can do:

```
/**
 * Views are the coupling of an AggregatorFactory applied to an Instrument.
 * Views are the connection to Accumulator and used to apply various processing to instruments before
 * storing the value in Accumulator.
 * */
```

Reply:
Hmm. Thinking on this more, I think we should probably just leave this as a todo until we have an actual specification for Views.

Reply:
done